module/platform/Webots/src/Webots.cpp

```diff
@@ -159,6 +159,71 @@ namespace module::platform {
         throw std::runtime_error(fmt::format("Unable to translate unknown NUgus.proto servo id: {}", id));
     }
 
+    [[nodiscard]] uint32_t translate_name_to_id(const std::string& name) {
+        if (name == "right_shoulder_pitch_sensor") {
+            return 0;
+        }
+        if (name == "left_shoulder_pitch_sensor") {
+            return 1;
+        }
+        if (name == "right_shoulder_roll_sensor") {
+            return 2;
+        }
+        if (name == "left_shoulder_roll_sensor") {
+            return 3;
+        }
+        if (name == "right_elbow_pitch_sensor") {
+            return 4;
+        }
+        if (name == "left_elbow_pitch_sensor") {
+            return 5;
+        }
+        if (name == "right_hip_yaw_sensor") {
+            return 6;
+        }
+        if (name == "left_hip_yaw_sensor") {
+            return 7;
+        }
+        if (name == "right_hip_roll_sensor") {
+            return 8;
+        }
+        if (name == "left_hip_roll_sensor") {
+            return 9;
+        }
+        if (name == "right_hip_pitch_sensor") {
+            return 10;
+        }
+        if (name == "left_hip_pitch_sensor") {
+            return 11;
+        }
+        if (name == "right_knee_pitch_sensor") {
+            return 12;
+        }
+        if (name == "left_knee_pitch_sensor") {
+            return 13;
+        }
+        if (name == "right_ankle_pitch_sensor") {
+            return 14;
+        }
+        if (name == "left_ankle_pitch_sensor") {
+            return 15;
+        }
+        if (name == "right_ankle_roll_sensor") {
+            return 16;
+        }
+        if (name == "left_ankle_roll_sensor") {
+            return 17;
+        }
+        if (name == "neck_yaw_sensor") {
+            return 18;
+        }
+        if (name == "head_pitch_sensor") {
+            return 19;
+        }
+
+        throw std::runtime_error(fmt::format("Unable to translate unknown NUgus.proto servo name: {}", name));
+    }
```

Comment:
```suggestion
    [[nodiscard]] uint32_t translate_name_to_id(const std::string& name) {
       
        std::vector<std::string> names{};
        
        // Fill names below, or alternatively declare somewhere else
        
        for (size_t i = 0; i < names.size(); ++i) {
            if (name == names[i]) {
                return i;
            }
        }
        throw std::runtime_error(fmt::format("Unable to translate unknown NUgus.proto servo name: {}", name));
    }
```
something like this instead? Alternatively do as a switch statement or use a map so it could just be one declaration within Webots.hpp and this function does not need to exist anymore as one can just do `uint8_t id = names[name];`

Reply:
or turn clang format off and have each if on one line

Reply:
Or if it's one specific file/folder that's got a lot of these you can put another `.clang-format` file in here with 
```yaml
BasedOnStyle: InheritParentConfig
# rules you want to change
```
and it'll then apply to everything in the folder the .clang-format is and all folders below it.